Makefile

```diff
@@ -126,6 +126,11 @@ gen-examples:
 gen-project: $(PYMODEL)
 	$(RUN) gen-project ${CONFIG_YAML} -d $(DEST) $(SOURCE_SCHEMA_PATH) && mv $(DEST)/*.py $(PYMODEL)
 
+# generate pandera schemas
+.PHONY: gen-pandera
+gen-pandera:
```

Comment:
why is this not in project.makefile?

Reply:
I missed that, cleaned up now, thankyou!